### PR TITLE
docs: Suggested technical writing changes for BYOB.

### PR DESCRIPTION
**Pull request by hayleycd**
@laurentsimon, @olivekl asked me to take a look at the BYOB docs. These are my suggested changes, and I am open to feedback.

### CODE REVIEW

**Review by laurentsimon — APPROVED** [review]
Thank you!

**Comment by laurentsimon** [comment]
The failing linter seems to jut be because the title and actual section names don't match

File: BYOB.md [hunk 1]
```diff
@@ -10,21 +10,21 @@
   - [SLSA GitHub Repository](#slsa-github-repository)
     - [SLSA Setup Action (SSA)](#slsa-setup-action-ssa)
     - [SLSA Reusable Workflow (SRW)](#slsa-reusable-workflow-srw)
-- [Integration Steps](#integration-steps)
-  - [Supported Triggers](#supported-triggers)
-  - [TRW inputs](#trw-inputs)
+- [Example: Convert a GitHub Action to be SLSA3 Compliant](#example-convert-a-github-action-to-be-slsa3-compliant)
+  - [Before you begin: understand supported triggers](#supported-triggers)
```
Comment (ianlewis) [inline BYOB.md:14]:
```suggestion
  - [Before you begin: understand supported triggers](#before-you-begin-understand-supported-triggers)
```

File: BYOB.md [hunk 2]
```diff
@@ -247,14 +257,16 @@ The last thing to do in the TCA is to [generate the metadata layout file](https:
...
+You may want to [return the attestation to the PW](https://github.com/laurentsimon/byob-doc/tree/v0.0.1/.github/workflows/builder_example_slsa3.yml#L69-L75) in case end-users want to publish the artifacts and attestations themselves. If you do so, we encourage you to create a [secure-download-attestation](https://github.com/laurentsimon/byob-doc/tree/v0.0.1/download/attestation/action.yml) Action for your users under a download folder in your repository. This will improve user experience as they won't have to be aware of the SLSA repository and its framework.
 
 ## PW Integration
 
+TODO: Provide a little context about what you mean by "PW integration"
```
Comment (ianlewis) [inline BYOB.md:268]: nit: hide this in a comment and add an issue number.

```suggestion
<!-- TODO(#2405): Provide a little context about what you mean by "PW integration" -->
```

File: BYOB.md [hunk 3]
```diff
@@ -311,7 +323,7 @@ The next thing to do is to [_not_ uploads asset to the GitHub release](https://g
 
 #### Update TRW
 
-Now we need to download the artifact and publish it from the TRW. Do do that, follow the steps:
+Now we need to download the artifact and publish it from the TRW. To do that, follow the steps:
```
Comment (ianlewis) [inline BYOB.md:326]:
```suggestion
Now we need to download the artifact and publish it from the TRW. To do that, follow these steps:
```

**Pull request merged**